src/measure_theory/function/strongly_measurable/ae_sequence.lean

```diff
@@ -0,0 +1,120 @@
+/-
+Copyright (c) 2023 Alex Kontorovich and Heather Macbeth. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Alex Kontorovich, Heather Macbeth
+-/
+
+import measure_theory.function.strongly_measurable.basic
+
+/-!
+# Sequence of strongly measurable functions associated to a sequence of a.e.-strongly-measurable
+  functions
+
+We define here tools to prove statements about limits (infi, supr...) of sequences of
+`ae_strongly_measurable` functions.
+Given a sequence of a.e.-strongly-measurable functions `f : ι → α → β` with hypothesis
+`hf : ∀ i, ae_strongly_measurable (f i) μ`, and a pointwise property `p : α → (ι → β) → Prop` such
+that we have `hp : ∀ᵐ x ∂μ, p x (λ n, f n x)`, we define a sequence of strongly measurable functions
+`ae_strongly_seq hf p` and a measurable set `ae_strongly_seq_set hf p`, such that
+* `μ (ae_strongly_seq_set hf p)ᶜ = 0`
+* `x ∈ ae_strongly_seq_set hf p → ∀ i : ι, ae_strongly_seq hf hp i x = f i x`
+* `x ∈ ae_strongly_seq_set hf p → p x (λ n, f n x)`
+## Implementation notes: This file follows closely the construction of `ae_measurable_sequence.ae_seq`, adapting it to the `ae_strongly_seq` setting.
```

Comment:
This renders very poorly, see https://observablehq.com/@bryangingechen/github-lean-doc-preview?url=%22https%3A%2F%2Fgithub.com%2Fleanprover-community%2Fmathlib%2Fpull%2F18380%22#docs. We don't want the whole note in the heading!